### PR TITLE
Manifest dev file

### PR DESCRIPTION
**Pull request by pallavi-udmalpet-sfdc**

### CODE REVIEW

File: aws/dev/pallavi-us-east-2.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  image: 985884022631.dkr.ecr.us-east-2.amazonaws.com/cni-nginx-proxy-outbound
+  version: '9'
+enable_sitebridge: false
+enable_transit_gateway: false
+outbound_vpcs_config:
+  1:
+    vpc_cidr: 10.20.8.0/22
+  2:
+    vpc_cidr: 10.20.12.0/22
+inbound_service_name: stage1-useast2-cni-nginx-inbound
```
Comment (kevinwang-sfdc) [inline aws/dev/pallavi-us-east-2.yaml:50]: We no longer use `inbound_service_name` and `outbound_service_name`, so you can remove them.

File: aws/dev/pallavi-us-east-2.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    # Monitoring VM
+    - 44.233.159.220/32
+lambda_layer_s3_key: zip/cni_common_v226-9.zip
+lambda_function_s3_key: zip/api_v5.zip
+lambda_memory_size: 2048
+
+eks_k8s_version: '1.15'
+inbound_vpc_cidr: 10.20.8.0/22
+inbound_data_plane_node_group_instance_types:
+  - m5.xlarge
```
Comment (agajjala) [inline aws/dev/pallavi-us-east-2.yaml:24]: @pallavi-udmalpet-sfdc can we go with `t3.medium` for DEV Stacks..

File: aws/dev/pallavi-us-east-2.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+api_authorization: CUSTOM
+api_authorizer_gdot_url: https://login.stmpa.stm.salesforce.com/services/c2c/api/v1/validateJwt
+api_authorizer_c2c_key_secret_name: C2C-EC-Keys-Dev
+api_access_whitelist_ips:
+    # Monitoring VM
+    - 44.233.159.220/32
+lambda_layer_s3_key: zip/cni_common_v226-9.zip
+lambda_function_s3_key: zip/api_v5.zip
+lambda_memory_size: 2048
```
Comment (agajjala) [inline aws/dev/pallavi-us-east-2.yaml:19]: Also `1024` for DEV stacks

File: aws/dev/dev-us-east-2-pal456.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+tags:
+  region: us-east-2
+  env_name: dev
+  deployment_id: pal456
+private_connect_role_name: dev-pal456-us-east-2-private-connect
+
+api_authorization: CUSTOM
+api_authorizer_gdot_url: https://login.stmpa.stm.salesforce.com/services/c2c/api/v1/validateJwt
+api_authorizer_c2c_key_secret_name: C2C-EC-Keys-Dev
+api_access_whitelist_ips:
```
Comment (agajjala) [inline aws/dev/dev-us-east-2-pal456.yaml:14]: remove line 14-16

**Pull request merged**